tests/datasource/fluent/test_spark_azure_blob_storage_datasource.py

```diff
@@ -333,102 +243,16 @@ def test_csv_asset_with_non_string_batching_regex_named_parameters(
     mock_list_keys.return_value = object_keys
     asset = spark_abs_datasource.add_csv_asset(
         name="csv_asset",
-        batching_regex=r"(.+)_(.+)_(?P<price>\d{4})\.csv",
         abs_container="my_container",
     )
     with pytest.raises(ge_exceptions.InvalidBatchRequestError):
         # price is an int which will raise an error
         asset.build_batch_request({"name": "alex", "timestamp": "1234567890", "price": 1300})
 
 
-@pytest.mark.big
-@pytest.mark.xfail(
-    reason="Accessing objects on azure.storage.blob using Spark is not working, due to local credentials issues (this test is conducted using Jupyter notebook manually)."  # noqa: E501
-)
-def test_get_batch_list_from_fully_specified_batch_request(
-    monkeypatch: pytest.MonkeyPatch,
-    spark_abs_datasource: SparkAzureBlobStorageDatasource,
-):
-    azure_client: azure.BlobServiceClient = cast(azure.BlobServiceClient, MockBlobServiceClient())
-
-    def instantiate_azure_client_spy(self) -> None:
-        self._azure_client = azure_client
-
-    monkeypatch.setattr(
-        great_expectations.execution_engine.sparkdf_execution_engine.SparkDFExecutionEngine,
-        "_instantiate_s3_client",
-        instantiate_azure_client_spy,
-        raising=True,
-    )
-    asset_specified_metadata = {"asset_level_metadata": "my_metadata"}
-    asset = spark_abs_datasource.add_csv_asset(
-        name="csv_asset",
-        batching_regex=r"(?P<name>.+)_(?P<timestamp>.+)_(?P<price>\d{4})\.csv",
-        abs_container="my_container",
-        batch_metadata=asset_specified_metadata,
-    )
-
-    request = asset.build_batch_request({"name": "alex", "timestamp": "20200819", "price": "1300"})
-    batches = asset.get_batch_list_from_batch_request(request)
-    assert len(batches) == 1
-    batch = batches[0]
-    assert batch.batch_request.datasource_name == spark_abs_datasource.name
-    assert batch.batch_request.data_asset_name == asset.name
-    assert batch.batch_request.options == {
-        "path": "alex_20200819_1300.csv",
-        "name": "alex",
-        "timestamp": "20200819",
-        "price": "1300",
-    }
-    assert batch.metadata == {
-        "path": "alex_20200819_1300.csv",
-        "name": "alex",
-        "timestamp": "20200819",
-        "price": "1300",
-        **asset_specified_metadata,
-    }
-    assert batch.id == "spark_abs_datasource-csv_asset-name_alex-timestamp_20200819-price_1300"
-
-    request = asset.build_batch_request({"name": "alex"})
-    batches = asset.get_batch_list_from_batch_request(request)
-    assert len(batches) == 2
```

Comment:
This test fails with a nasty spark error - it wasn't immediately clear to me if the error was credentials as indicated in the `pytest.mark.skip`, but it also looks like the test was never completed so i'm removing.